config/v3_0/types/config.go

```diff
@@ -24,3 +30,49 @@ var (
 		Minor: 0,
 	}
 )
+
+func (cfg Config) Validate(c path.ContextPath) (r report.Report) {
+	systemdPath := "/etc/systemd/system/"
+	unit_paths := make([]string, 0)
```

Comment:
If this is a set (i.e. `map[string]struct{}`) we can avoid iterating over it during lookups.

Reply:
Agreed. That's a much better approach.